integrations/categories.yaml

```diff
@@ -0,0 +1,267 @@
+deploy:
+  - id: operating-systems
+    parent: deploy
+    name: Operating Systems
+    description: ""
+    most-popular: false
+  - id: docker-kubernetes
+    parent: deploy
+    name: Docker & Kubernetes
+    description: ""
```

Comment:
I think we need to separate categories because Docker and K8s have different deploy instructions.

Reply:
that's another thing that is grey on testing, shouldn't k8s and docker be cards inside that category? (at the moment that category is empty, was told to add those categories)

Reply:
Because then you would have only one card 
![image](https://github.com/netdata/netdata/assets/70198089/00b3bcd5-6601-47cf-8af9-b9798f11c546)

for k8s inside Kubernetes

Reply:
> then you would have only one card

You are right!